Week3/js-exercises-week3/takeOutTheDuplicates.js

```diff
@@ -0,0 +1,10 @@
+const letters = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c', 'b'];
+
+
+function removeDuplicates(array) {
+    let uniqArr = Array.from(new Set(array))
+    return uniqArr
+}
```

Comment:
Cool way that gets the job done. If this was a real project I would say kudos and accept this solution.
But for the purpose of this week's exercise you need to achieve the same using closures and without using the Set object. Similar to what we did in the class